administrator-guides/setting-up-video-conferencing/README.md

```diff
@@ -0,0 +1,40 @@
+# Video Conferencing
+
+If you want to enable video conferencing, you have 2 options, [Jitsi Meet](#jitsi-meet) or [RocketChat Meet](#rocketchat-meet-webrtc).
+
+We recommend using Jitsi for video conference, since RocketChat Meet is still a work in progress.
+
+## Jitsi Meet
+
+You can use the [Jitsi Meet](https://jitsi.org/jitsi-meet/) video conferencing platform embedded in Rocket.Chat.
+
+The [Jitsi Meet prject](https://jitsi.org/Projects/JitsiMeet) (Jitsi Video Bridge) is a tried and true bandwidth efficient WebRTC compatible SFU (server based)
+solution from our gracious FOSS partner, [Jitsi](https://jitsi.org/).
+
+Through the collaboration arrangement with Jitsi, Rocket.Chat users can enjoy reliable and robust
+group video chat, audio chat, and screen sharing experience out of the box.
+
+To enable Jitsi:
+
+Go to `Administation -> SETTINGS -> Video Conference` then set `Enabled`  to `True`.
+
+From there you can set where video conferences can be held (channels, private rooms and direct messages).
+
+### How to set up your own Jitsi Meet server
+
+In some use-cases, where the Jitsi.org infrastructure may be remote or inaccessible, or where you
+require total privacy, setting up your own Jitsi Meet servers stack may be necessary.
+
+You can find extensive instructions on setting up a Jitsi Meet servers stack at the [official Jisti Meet repository](https://github.com/jitsi/jitsi-meet).
+
+Also, please consult our guide to quick setup of Jitsi Meet servers stack via Docker image.
```

Comment:
is there a guide / page that we can link to ?  if not, perhaps remove this line.